src/main.cpp

```diff
@@ -1,7 +1,20 @@
 #include <stdio.h>
+#include <vector>
+#include <memory>
 
+#include "core.h"
 
 int main(int argc, char** argv) {
-    printf("nothing yet\n");
+    printf("creating CVC\n");
+    std::unique_ptr<DecisionEngine> d;
+    std::vector<std::unique_ptr<Character>> c;
+    c.push_back(std::move(std::unique_ptr<Character>()));
```

Comment:
this seems... verbose, is it right?

Reply:
yeah, should do `c.push_back(std::make_unique<Character());`

Reply:
You don't need the std::move, because the temporary is already an r-value.